services/metadactyl-clj/src/metadactyl/routes/tools.clj

```diff
@@ -286,159 +287,159 @@
          :description "This service submits a request for a tool to be installed so that it can be used
          from within the Discovery Environment. The installation request and all status updates
          related to the tool request will be tracked in the Discovery Environment database."
-         (service/trap uri submit-tool-request current-user body))
+         (ok (submit-tool-request current-user body)))
 
-  (GET* "/status-codes" [:as {uri :uri}]
+  (GET* "/status-codes" []
         :query [params StatusCodeListingParams]
         :summary "List Tool Request Status Codes"
         :return StatusCodeListing
         :description "Tool request status codes are largely arbitrary, but once they've been used once,
         they're stored in the database so that they can be reused easily. This endpoint allows the
         caller to list the known status codes."
-        (service/trap uri list-tool-request-status-codes params)))
+        (ok (list-tool-request-status-codes params))))
 
 (defroutes* admin-tools
-  (POST* "/" [:as {uri :uri}]
+  (POST* "/" []
          :query [params SecuredQueryParams]
          :body [body (describe ToolsImportRequest "The Tools to import.")]
          :summary "Add new Tools."
          :description (str "This service adds new Tools to the DE." volumes-from-warning)
-         (ce/trap uri #(add-tools body)))
+         (add-tools body))
```

Comment:
For real this time, I assume this actually needs an `(ok ...)`, since you removed the `success-response` call from `metadactyl.tools/add-tools` (correctly). Presumably also true for `update-tool` and `delete-tool-device` below.